stdlib/printexc.mli

```diff
@@ -277,6 +285,12 @@ val convert_raw_backtrace_slot : raw_backtrace_slot -> backtrace_slot
 *)
 
 
+val get_raw_backtrace_next_slot : raw_backtrace -> raw_backtrace_slot option
+(** [get_raw_backtrace_next_slot slot] returns the next slot inlined, if any.
+
```

Comment:
The problem is that the type is wrong, it should be `val get_raw_backtrace_next_slot : raw_backtrace_slot -> raw_backtrace_slot option`! My bad.

The pattern is that `get_raw_backtrace_slot` returns the head of the debuginfo linked-list and then repeated calls to `get_raw_backtrace_next_slot` allow to traverse the list.